src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -35,7 +33,9 @@ import (
 
 const (
 	SrcAuthfilePath = "/root/containers/auth.json"
+	AaFilePath      = "/run/peerpod/aa.toml"
```

Comment:
nit: filepath should be redundant, AAConfigPath or AAConfigFile wiuld be better and unambiguous imo AA also has a socket "file".

Reply:
Makes sense to me.